src/corelibs/U2View/src/ov_sequence/AnnotationsTreeView.cpp

```diff
@@ -1607,9 +1609,14 @@ void AnnotationsTreeView::sl_itemClicked(QTreeWidgetItem* i, int column) {
     if (lastMB != Qt::LeftButton || item == nullptr || !item->isColumnLinked(column)) {
         return;
     }
+
     QString fileUrl = item->getFileUrl(column);
     if (!fileUrl.isEmpty()) {
-        Task* task = new LoadRemoteDocumentAndAddToProjectTask(fileUrl);
```

Comment:
There was a contract in the AnnotationsTreeView: if an item contains an `fileUrl` it points to the downloadable resource.
It is does not matter where this URL comes from: a DbXRef subsystem or some other source.

With this change you break this abstraction and force the URL to come from DbXref only.
The original behavior should be preserved: if you have an URL the url should be downloaded and opened as is.

I see that in this case you may loose the relation of the URL to the DB and do not know how to generate the report (the goal of the task).

This can be done differently: build extra information about the URL: 
1) either by parsing & matching the URL inside of the report generator 
or
2) by passing additional report metadata to the loader task

Reply:
> There was a contract in the AnnotationsTreeView: if an item contains an fileUrl it points to the downloadable resource.

Could you, please, provide an exact scenario of this. There is no GUI tests for this scenario, also I tried to edit qualifiers by my own and failed in reproducing, Without scenario I can't get what is wrong about it. If you provide a scenario, I will fix and and will add a GUI test.